### PR TITLE
config: add graph `2023-04-13`

### PR DESCRIPTION
**Pull request by teowa**
swagger: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/graphservicesprod/resource-manager/Microsoft.GraphServices/stable/2023-04-13/graphservicesprod.json

### CODE REVIEW

**Comment by tombuildsstuff** [comment]
@teowa is this Microsoft Graph? What's this provisioning? cc @manicminer

**Comment by teowa** [comment]
Now only the Microsoft.GraphServices/accounts resource can be provisioned. From [doc](https://learn.microsoft.com/en-us/graph/metered-api-setup?tabs=azurecloudshell#enable-an-application):
> To enable an application to use metered APIs and services in Microsoft Graph, it must be associated with an Azure subscription. To create this association, you must create an Azure resource of type Microsoft.GraphServices/accounts. The Azure resource connects a single Azure AD application registration with the Azure subscription where the application's usage of metered APIs is billed.

**Comment by manicminer** [comment]
@teowa Thanks for suggesting to add this. In general we'd prefer to keep that functionality in the AzureAD TF provider (via MS Graph). Is there functionality available in this RP that is not available by directly querying Microsoft Graph?

**Review by manicminer — APPROVED** [review]

File: config/resource-manager.hcl [hunk 1]
```diff
@@ -223,6 +223,10 @@ service "frontdoor" {
   name      = "FrontDoor"
   available = ["2020-04-01", "2020-05-01", "2022-05-01"]
 }
+service "graphservicesprod" {
```
Comment (manicminer) [inline config/resource-manager.hcl:226]:
```suggestion
service "graphservices" {
```

**Pull request merged**